crates/conjure_core/src/rules/bubble.rs

```diff
@@ -75,6 +75,17 @@ fn div_to_bubble(expr: &Expression, _: &Model) -> ApplicationResult {
         return Err(RuleNotApplicable);
     }
     if let Expression::UnsafeDiv(_, a, b) = expr {
+        // bubble bottom up
+        if a.can_be_undefined() || b.can_be_undefined() {
+            return Err(RuleNotApplicable);
+        }
+
+        // only do one bubble at a time
+        if matches!(**a, Expression::Bubble(_, _, _)) || matches!(**b, Expression::Bubble(_, _, _))
```

Comment:
we should be able to do multiple bubbles at once too.

```
{a @ bubble_a} + {b @ bubble_b}

~~> bubble-up
{a + b @ bubble_a /\ bubble_b}
```

is a valid transformation. 

I know this rule doesn't do `+` but I noticed the check here, so noting here.

Reply:
in fact the `bubble_up` rule already does this. ignore.